.github/workflows/update-contribs.yml

```diff
@@ -0,0 +1,44 @@
+name: Update Contributors
+
+on:
+  pull_request:
+  push:
+    branches:
+      - main
+jobs:
+    run-meta:
+        runs-on: ubuntu-latest
+        steps:
+        - name: Check out the code
+          uses: actions/checkout@v3
+        - name: Setup Python
+          uses: actions/setup-python@v2
+          with:
+            python-version: '3.10'
+        - name: Upgrade pip
+          run: |
+            # install pip=>20.1 to use "pip cache dir"
+            python3 -m pip install --upgrade pip
+            echo "$PWD"
+            git branch
```

Comment:
```suggestion
```

The `echo $PWD` is already run just below (with nothing that could change the CWD in between), and you removed the branch creation step before this (and `create-pull-request` will print it anyway), so it seems like these are both vestigial. And just in case you do need it, it seems the next step would be a more logical place for it.

Reply:
ok! this was added because i didn't understand why i wasn't seeing changes / a pr from this action. i am still a bit confused about how the branch part of this action works.

Reply:
Hopefully the explanation I DM'ed was helpful, but if not happy to explain further or hop on a quick call.